legiti/src/main/java/com/legiti/LegitiClient.kt

```diff
@@ -23,8 +24,28 @@ class LegitiClient : LegitiService {
         this.androidContext = null
     }
 
+
+    @Deprecated(
+        message = "This function is deprecated and will be removed in a future version of the library",
+        level = DeprecationLevel.WARNING,
+        replaceWith = ReplaceWith(
+            expression = "Legiti.sharedInstance().setup(authToken)",
+            imports = ["com.legiti.Legiti"]
+        )
+    )
     override fun setup(authToken: String, legitiDevEnv: Boolean) {
```

Comment:
do we need to add the log statement in this function like we did here? https://github.com/legiti/legiti-android/pull/33/files#diff-3e77860bff2b06b1873ea0e49ffc5ea4R127

Reply:
I was thinking about this, but since I added the `ReplaceWith` I think we don't need it since androidStrudio will show the option to replace the function.

Reply:
hmm if you don't mind, let's just add the log statement anyway? I don't see any downside